src/renderer/src/components/select-folder/index.ts

```diff
@@ -0,0 +1 @@
+export * from './SelectFolder';
```

Comment:
Hmm we need to figure out exports

Reply:
yep, It seems like the eslint rule didn't take place on Github actions